source/includes/administration/_pricings.md

```diff
@@ -238,6 +238,21 @@ Required | &nbsp;
 `cogs`<br/>*Map* | Map from currency code to cogs
 `unitPrice`<br/>*Map* | Map from currency code to unit price
 
+Optional | &nbsp;
+---- | -----------
+`pricingTiers` <br/>*Array<PricingTier>* | List of pricing tiers on priced product
```

Comment:
The attributes should be denoted as Javascript primitives/objects or custom objects referenced elsewhere in the documentation (Organization, ServiceConnection, etc.).

I don't think there's a `PricingTier` object defined in our docs. This should be a `Array[Object]` and the keys on them should be documented.

Reply:
For example, `Array[[Role](#administration-roles)]`.

Reply:
ahhh okay I was gonna ask you @dpmurray21 about this but forgot... why is it that we don't define pricing tier as an object but we have something like PricedProduct as one?

Reply:
It looks like the `PricedProduct` is defined afterwards in another table. I guess you can do something similar. I prefer this approach to having the keys in the same table tbh.

```
PricedProduct

Attributes | &nbsp;
---- | -----------
`product.id`<br/>*UUID* | The id of the product to add prices.
`cogs`<br/>*Map<string, double>* | Map from currency to price. Must include all currencies in pricing at that point (see Effective pricing API).
`unitPrice`<br/>*Map<string, double>* | Map from currency to price. Must include all currencies in pricing at that point (see Effective pricing API).
```

Reply:
like you did below already 😛